rustls/src/error.rs

```diff
@@ -1,11 +1,40 @@
-use crate::msgs::enums::{AlertDescription, ContentType, HandshakeType};
+use crate::msgs::enums::{AlertDescription, ContentType, HandshakeType, KeyUpdateRequest};
 use crate::rand;
 
 use std::error::Error as StdError;
 use std::fmt;
 use std::sync::Arc;
 use std::time::SystemTimeError;
 
+/// A corrupt TLS message payload that resulted in an error.
+#[non_exhaustive]
+#[derive(Debug, Clone)]
```

Comment:
I think we want to derive `Copy` here, too.

Reply:
Sure, I can do that if you'd like. I originally withheld from doing so since I was matching the traits implemented by `PeerIncompatible`, which only has `Clone`